Packs/FortiSandbox/Integrations/FortiSandboxv2/FortiSandboxv2.py

```diff
@@ -0,0 +1,1182 @@
+    else:  # urls:
+        file_name = f"urls_for_upload_{time.time()}.txt"
+        content = "\n".join(urls)
+
+    return b64_encode(file_name), b64_encode(content)
+
+
+def poll_job_submissions(client: Client, args: dict[str, Any]) -> PollResult:
+    """Poll for the verdicts/reports of the given submission job IDs.
+
+    Args:
+        client (Client): Session to the API to run HTTP requests.
+        args (dict[str, Any]): Arguments passed down by the CLI to configure the request.
+
+    Returns:
+        PollResult: The results of the polling, includes verdicts within CommandResults or fileResults.
+    """
+    sid = args["sid"]
+    get_scan_report = argToBoolean(args.get("get_scan_report", False))
+
+    integration_context = get_integration_context()
+    total_jids: int = integration_context.get("total_jids", 0)
+    fetched_jids = set(integration_context.get("fetched_jids", []))
+    remaining_jids = set(integration_context.get("remaining_jids", []))
+    jid_to_raw_response: dict[str, dict[str, Any]] = integration_context.get("jid_to_raw_response", {})
+
+    # Enter if the total number of jobs is not known yet, or if not all jobs were created.
+    if not total_jids or (len(remaining_jids) + len(fetched_jids)) < total_jids:
+        raw_response = client.submission_job_list(sid)
+
+        raw_response = dict_safe_get(raw_response, ["result", "data"], {})
+        jids = set(raw_response.get("jids", []))
+        total_jids = raw_response.get("total_jids", 0)
+
+        remaining_jids = jids - fetched_jids if fetched_jids else jids
+
+    # Poll for the verdicts/reports of the remaining jobs.
+    for jid in remaining_jids.copy():
+        raw_response = client.submission_job_verdict(jid)
+        data = dict_safe_get(raw_response, ["result", "data"])
+
+        # If a result was fetched, add it to the list of results and remove it from the list of remaining jobs.
+        if data:
+            fetched_jids.add(jid)
+            remaining_jids.remove(jid)
+            jid_to_raw_response[jid] = raw_response
+
+    # Update the integration context with the new data for the next polling.
+    set_integration_context(
+        {
+            "total_jids": total_jids,
+            "fetched_jids": list(fetched_jids),
+            "remaining_jids": list(remaining_jids),
+            "jid_to_raw_response": jid_to_raw_response,
+        }
+    )
+
+    fetched_jids_count = len(fetched_jids)
+
+    # If not all jobs were fetched, continue to poll.
+    if not total_jids or fetched_jids_count < total_jids:
+        return PollResult(
+            response=None,
+            continue_to_poll=True,
+            args_for_next_run=args,
+            partial_result=CommandResults(
+                readable_output=(
+                    f"## {fetched_jids_count} out of {total_jids} jobs were fetched for the submission {sid}."
+                    if total_jids
+                    else f"## No jobs were created yet for the submission {sid}."
```

Comment:
How come, this will only print if we have 0 in `total_jobs`, otherwise we will start the count.
I can remove this and change to the default value, but wouldn't it be more informative to handle the commands with this output?

Reply:
lets leave this